arian1917/arian1917/Program.cs

```diff
@@ -0,0 +1,579 @@
+﻿using System.Data;
+using System.Diagnostics.Metrics;
+using System.Globalization;
+using System.Linq.Expressions;
+using System.Numerics;
+using System.Reflection.Metadata.Ecma335;
+using System.Runtime.CompilerServices;
+using Microsoft.Data.Sqlite; 
+
+
+namespace habitTracker{
```

Comment:
Use file-scoped namespaces to save an indentation level.
Also, Namespaces should be in [Pascal Case](https://www.theserverside.com/answer/Pascal-case-vs-camel-case-Whats-the-difference).

```
// Block-Scoped Namespace
namespace HabitTracker
{
    // Code here.
}
```
vs
```
// File-Scoped Namespace
namespace HabitTracker;
// Code here.
```